tests/standalone/PyQt5SSLSupport.py

```diff
@@ -0,0 +1,25 @@
+#     Copyright 2019, Kay Hayen, mailto:kay.hayen@gmail.com
+#
+#     Python test originally created or extracted from other peoples work. The
+#     parts from me are licensed as below. It is at least Free Software where
+#     it's copied from other people. In these cases, that will normally be
+#     indicated.
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+#
+import sys
+from PyQt5.QtNetwork import QSslSocket
+
+has_ssl_ssuport = QSslSocket.supportsSsl()
+print("SSL support: %r" % (has_ssl_ssuport,))
+sys.exit(not (has_ssl_ssuport is True))
```

Comment:
Should I just use `assert QSslSocket.supportsSsl()` to ensure the support is guaranted?

Reply:
No, actually the `print` is just perfect by itself. It will make sure that there is the same level of SSL support for both the Qt in CPython and the one in the standalone Nuitka. Test will fail with a diff already.

An exit code or an exception is not needed or wanted, because you are not supposed to test that PyQt has SSL support, only if it behaves the same way.

And by default, we make tests exit with 0, just so we know when they are fundamentally broken.

For top level import stuff, we tend to check in `run_all.py` if the relevant package, PyQt5 or so is even installed. Often, e.g. when running to package stuff, it is not.

Reply:
Actually "expect_success" is one of the options for tests. But we do not expect every Python to have every feature, esp. not if it's optional given there is a query function. :)

Reply:
Alright, I updated the test :)